inference-engine/tests_deprecated/unit/engines/gna/gna_graph_aot_test.cpp

```diff
@@ -44,11 +44,6 @@ TEST_F(GNAAOTTests, DISABLED_AffineWith2AffineOutputs_canbe_export_imported) {
 
 // Disabled because of random fails: CVS-23611
 TEST_F(GNAAOTTests, DISABLED_AffineWith2AffineOutputs_canbe_imported_verify_structure) {
-
-#if GNA_LIB_VER == 2
-    GTEST_SKIP();
```

Comment:
The test is disabled, have you checked if it works for GNA2? Shouldn't we switch it on now?

Reply:
done